src/v2/lib/validators-v2.js

```diff
@@ -1,23 +1,38 @@
-const Joi = require('@hapi/joi')
-  .extend(require('@hapi/joi-date'));
+const Joi = require('joi').extend(require('@joi/date'));
+
+const nullDefault = val => {
```

Comment:
I don't think we need this function here. If the value is left out from the object passed to the schema for validation it will set it to null as it was on the deleted line 20